scripts/test_data/print_command_line.py

```diff
@@ -14,7 +14,7 @@
 
 import streamlit as st
 
-server = st.server.server.Server.get_current()  # type: ignore
+server = st.server.server.Server.get_current()
```

Comment:
I'm not sure why `mypy` just started complaining about this line, but apparently we're no longer using this `type: ignore`